src/rust/engine/fs/store/src/lib.rs

```diff
@@ -867,19 +867,15 @@ mod local {
   use futures::future::{self, Future};
   use hashing::{Digest, Fingerprint, EMPTY_DIGEST};
   use lmdb::Error::{KeyExist, NotFound};
-  use lmdb::{
-    self, Cursor, Database, DatabaseFlags, Environment, EnvironmentCopyFlags, EnvironmentFlags,
-    RwTransaction, Transaction, WriteFlags,
-  };
-  use log::{error, trace};
+  use lmdb::{self, Cursor, Database, RwTransaction, Transaction, WriteFlags};
+  use log::error;
   use sha2::Sha256;
+  use sharded_lmdb::ShardedLmdb;
   use std;
-  use std::collections::{BinaryHeap, HashMap};
-  use std::fmt;
-  use std::path::{Path, PathBuf};
+  use std::collections::BinaryHeap;
+  use std::path::Path;
   use std::sync::Arc;
   use std::time;
-  use tempfile::TempDir;
 
   use super::MAX_LOCAL_STORE_SIZE_BYTES;
```

Comment:
I'm a little bit confused as to why this lives here, is this constant `fs`-dependent?

Reply:
Moved some things around, added some comments